dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/NonPersistedDataManager.kt

```diff
@@ -0,0 +1,37 @@
+package org.dataland.datalandbackend.services
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.dataland.datalandbackend.model.StorageHashMap
+import org.dataland.datalandbackendutils.exceptions.ResourceNotFoundApiException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.stereotype.Component
+
+/**
+ * Implementation of a data manager for Dataland including metadata storages
+ * @param objectMapper object mapper used for converting data classes to strings and vice versa
+ * @param dataInformationHashMap temporarily datastore
+*/
+@ComponentScan(basePackages = ["org.dataland"])
+@Component("NonPersistedDataManager")
+class NonPersistedDataManager(
```

Comment:
this feels inconsistent: The DataManger stores things in the dataInformationHashMap, and the NonPersistedDataManager takes things out again.

Why not have both things in one class?

Reply:
Agreed. Done.

Reply:
ok